modules/Authentication.php

```diff
@@ -1,11 +1,17 @@
 <?php
 	class Authentication {
-		private $user;
+		public $user;
 
-		function __construct($email, $password) {
-			if (!($email && $password)) {
-				return false;
+		public function __construct($email, $password) {
+			if (!$email || !$password) {
+				return NULL;
```

Comment:
@MehaKaushik Check this https://stackoverflow.com/questions/175532/should-a-retrieval-method-return-null-or-throw-an-exception-when-it-cant-prod?answertab=active#tab-top